src/main/scala/cromwell/engine/WorkflowManager.scala

```diff
@@ -47,50 +65,79 @@ trait WorkflowManager {
 /**
  * A WorkflowManager using WorkflowActors as the workflow handler
  */
-trait ActorWorkflowManager extends WorkflowManager {
-  override type Workflow = ActorRef // TODO: In a world where Akka Typed is no longer experimental switch to that
+
+object ActorWorkflowManager {
+  def props: Props = Props(classOf[ActorWorkflowManager])
+}
+
+class ActorWorkflowManager extends WorkflowManagerActor {
   override val backend = new LocalBackend
-  implicit def actorSystem: ActorSystem
+  implicit val timeout = Timeout(30.seconds)
 
-  override def generateWorkflow(id: WorkflowId, wdl: WdlSource, inputs: WorkflowInputs): Try[Workflow] = {
+  override def generateWorkflow(id: WorkflowId, wdl: WdlSource, inputs: binding.WorkflowInputs): Try[Workflow] = {
     val binding = Try(WdlBinding.process(wdl))
-    binding map {x => actorSystem.actorOf(WorkflowActor.buildWorkflowActorProps(id, x, inputs))}
+    binding map {x => context.actorOf(WorkflowActor.buildWorkflowActorProps(id, x, inputs))}
   }
 
-  override def workflowStatus(id: WorkflowId): Option[WorkflowState] = workflowById(id) map {x => workflowState(x)}
+  override def workflowOutputs(id: WorkflowId): Future[Option[binding.WorkflowOutputs]] = {
+    workflowById(id) match {
+      case Some(x) =>
+        val eventualSymbolStoreEntries = {x ? GetOutputs}.mapTo[Set[SymbolStoreEntry]]
+        symbolsToWorkflowOutputs(eventualSymbolStoreEntries) map {Option(_)}
+      case None => Future {None}
+    }
+  }
 
-  override def submitWorkflow(wdl: WdlSource, inputs: WorkflowInputs): Try[ManagedWorkflow] = {
-    val managedWorkflow = super.submitWorkflow(wdl, inputs)
-    managedWorkflow foreach {_.workflow ! Start(backend)}
-    managedWorkflow
+  override def submitWorkflow(wdl: WdlSource, inputs: binding.WorkflowInputs): Future[Try[WorkflowId]] = {
+    val eventualManagedWorkflow = addWorkflow(wdl, inputs)
+    // Side effecting code to send this workflow a message to start, and then return the Future
+    eventualManagedWorkflow foreach {_ foreach {_.workflow ? Start(backend)}} // Needs to be an ask and not a tell as this isn't an Actor
+    eventualManagedWorkflow map {_ map {_.id}} // Convert the Future[Try[ManagedWorkflow]] to Future[Try[WorkflowId]]
```

Comment:
I can but I don't like it as there's now a side effect in the midst of the map's lambda. I think it's more clear that there's some side effecting code happening and then a transform